gitops/kcp/gateway-bindings/glbc-apibinding.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIBinding
+metadata:
+  annotations: {}
+  name: glbc
+spec:
+  permissionClaims:
+    - group: ""
+      resource: secrets
+      state: Accepted
+    - group: ""
+      # TODO: identityHash needs to match the specific value in the glbc apiexport of the cluster
```

Comment:
Am I understanding correctly that we should have a kustomization.yaml applied on top in the gitops SRE repository that would overwrite the identityHash with the correct value?

Reply:
Yes this is correct. see above